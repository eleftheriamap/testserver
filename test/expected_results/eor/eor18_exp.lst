
test/exp/test/gen/eor/eor18_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	91733442 	add	x2, x2, #0xccd, lsl #12
   4:	5157335a 	sub	w26, w26, #0x5cc, lsl #12
   8:	4a5a005c 	eor	w28, w2, w26, lsr #0
   c:	8a000000 	and	x0, x0, x0
