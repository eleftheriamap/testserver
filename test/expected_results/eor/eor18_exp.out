Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000ccd000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 00000000ffa34000
X27    = 0000000000000000
X28    = 00000000ff6f9000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x91733442
0x00000004 : 0x5157335a
0x00000008 : 0x4a5a005c
0x0000000c : 0x8a000000
